packages/merge-on-green/package-lock.json

```diff
@@ -0,0 +1,6446 @@
+{
```

Comment:
The other `package-lock.json` files were removed in this PR. Should this one be removed or the others restored?

Reply:
Yes, thank you for noticing!